src/test/java/dk/kb/datahandler/enrichment/DataEnricherTest.java

```diff
@@ -0,0 +1,105 @@
+package dk.kb.datahandler.enrichment;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dk.kb.datahandler.config.ServiceConfig;
+import dk.kb.datahandler.oai.OaiRecord;;
+import dk.kb.util.Resolver;
+import dk.kb.util.xml.XML;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class DataEnricherTest {
+
+    private FragmentsClient fragmentsClientMock;
+    private ObjectMapper mapper;
+
+    @BeforeEach
+    public void setUp() throws IOException, URISyntaxException {
+        ServiceConfig.initialize("conf/ds-datahandler-behaviour.yaml","conf/ds-datahandler-local.yaml");
```

Comment:
My guess is that you've made some configuration changes in your local config. For testing purposes could these be moved to aegis in the file: `ds-datahandler/local/src/test/resources/ds-datahandler-integration-test.yaml` or another test-config file placed in aegis. To see how this has been implemted have a look at the `setup`-method in datahandlers `OaiHarvestClientIntegrationTest`-class.

Reply:
I see that you've removed all configuration from the test class. That's pretty!